src/sections/Community/Handbook/connect.js

```diff
@@ -16,37 +23,96 @@ const Connect= () => {
-            <p>Join our <a href="http://slack.layer5.io/">Slack Workspace</a> and interact with people.
-       In the workspace, there are different channels to engage in. Here’s a description of them:</p>
-       
-            <p><span className="highlight" >#announcements</span> - For general announcements, projects, releases, security vulnerabilities.</p>
-            <p><span className="highlight">#buzz</span> - For notifications on Twitter activities.</p>
-            <p><span className="highlight">#careers</span> - For various career opportunities available </p>
-            <p><span className="highlight">#github-notifications</span> - For updates on Layer5 GitHub project activities</p>
-            <p><span className="highlight">#general</span> - For workspace-wide communication and announcements.</p>
-            <p><span className="highlight">#meshery</span> - For discussion on topics in and around Meshery</p>
-            <p><span className="highlight">#meshery-adapters</span> - Discuss topics related to meshery-adapters.</p>
-            <p><span className="highlight">#meshery-ci</span> - For discussion on topics related to meshery-ci</p>
-            <p><span className="highlight">#meshery-meshsync</span> - For discussions on issues related to mesh sync</p>
-            <p><span className="highlight">#mesheryctl</span> - For discussions on Meshery’s CLI</p>
-            <p><span className="highlight">#newcomers</span> - For welcoming first-time contributors and community newcomers.</p>
-            <p><span className="highlight">#osom</span> - For open-source software Mexico discussion</p>
-            <p><span className="highlight">#performance</span> - For discussions related to measuring and evaluating service mesh performance.</p>
-            <p><span className="highlight">#random</span> - For non-work banter and water cooler conversation.</p>
-            <p><span className="highlight">#smi</span> - For discussions related to Meshery and Service Mesh Interface (SMI).</p>
-            <p><span className="highlight">#support</span> - For support-related issues such as setup issues</p>
-            <p><span className="highlight">#wasm</span> - For discussions on web assembly, rust, data plane filters</p>
-            <p><span className="highlight">#websites</span> - For discussions related to layer5.io site.</p>
-            <p><span className="highlight">#workshops</span> - Q-A for workshops given at KubeCon, DockerCon, Velocity, O’Reilly Live Training</p>
+            <p>Join our <img height="40 px" src={slack_icon} alt="slack" /> <a href="http://slack.layer5.io/">Slack Workspace</a> “introduce yourself” and interact with the community. There are different channels in the workspace, to engage in, just search for:</p>
+            <p><a href="https://layer5io.slack.com/archives/CDM0ACDM5">#general</a> - For workspace-wide communication and announcements.</p>
```

Comment:
Can you try indenting this following list of channels slightly

Reply:
Done!!